### ISSUE TITLE
ModuleNotFoundError numpy 

### ISSUE THREAD

**Issue by lgratiela** [KratosMultiphysics/Kratos#4575]
Hi everybody! I am a newbie here and I'm trying to run a case with the Conv-Diff app (no problem there) but I keep getting errors when importing different libs. 

I managed to compile the HDF5 app only after specifying in the configure the paths for HDF5_LIBRARIES and HDF5_INCLUDE_DIRS and now it seems Kratos can't find numpy even though I have it installed (checked it with pip freeze). What am I missing? Is there something I should add to the system path? I tried adding %PYTHON_HOME%\Lib\site-packages\numpy\ but still it doesn't work. Help!!

**Comment by lgratiela** [comment]
@RiccardoRossi @roigcarlo

**Comment by roigcarlo** [comment]
Hi @lgratiela 

In order to use numpy in your scripts you need to compile kratos yourself (I understand that you already done that) and use python instead of runkratos to run the scripts. Currently we do not support numpy as a part of runkratos.

You should not touch the `PYTHON_HOME` variable under any circumstance for kratos to work.

Also take into account that you may have different versions of python installed, and the `pip frezee` will only show the installed package of the default one, which my be different from the one used to compile kratos.

**Comment by lgratiela** [comment]
Hi @roigcarlo! I checked if the default version of python is the one used for compiling Kratos and it is, so that's good. When running the script for the case, I run it with python MainKratos_Modified_.py but then it doesn't find Kratos ->  ModuleNotFoundError: No module named 'KratosMultiphysics' . I'm guessing I should add the Kratos folder to the system path, right?

**Comment by roigcarlo** [comment]
Yes. In case that error appear you have to add kratos folder to the `PYTHONPATH` and kratos/libs to the system `PATH` ( or `LD_LIBRARY_PATH` in linux )

**Comment by lgratiela** [comment]
Done! worked like a charm! ty @roigcarlo
